.github/workflows/build-and-test.yml

```diff
@@ -131,32 +131,29 @@ jobs:
       uses: docker/build-push-action@v5
       with:
         context: .
-        load: true
-        tags: ${{ env.TEST_IMAGE_TAG }}
+        push: true
+        tags: ort:run-${{ env.GITHUB_RUN_ID }}
```

Comment:
I think you have to use `${{ github.run_id }}` or maybe `$GITHUB_RUN_ID`, this page says those variables are not available from the `env` context: https://docs.github.com/de/actions/learn-github-actions/variables#default-environment-variables